drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc

```diff
@@ -0,0 +1,39 @@
+
+What does it mean to be cloud native and why is it important? Cloud native tools are purpose-built for the cloud. They take advantage of access to instantly available compute and storage resources that can be adapted quickly to the needs of the tool. Cloud native tools also offer simplified maintenance, resiliency, and elasticity, which gives you the ability to scale resources as needed.
+
+Introducing Tekton, a new open source project that provides a framework for creating cloud native CI/CD pipelines. Tekton benefits from a robust community of contributors and provides a set of fundamental building blocks for creating a CI/CD environment. Tekton provides a first-class Kubernetes integration, including custom resource definitions (CRDs) as a standard way to run end-to-end pipelines. 
+
+Each Tekton pipeline is made up of a sequence of tasks. For example, tasks could include a Git clone, a build, a vulnerability scan, running tests, or deploying the application. These tasks can be as simple or complex as needed. Tasks are tied together to make up a pipeline and can be run sequentially, concurrently, or on different nodes. Developers can create new tasks and share them across their enterprise by putting task containers into a container registry and consuming these containers in Tekton Tasks. This basic use case shows how leveraging a cloud native approach can simplify tool distribution across CI/CD systems and have an instant worldwide distribution of this new tool. Teams no longer need to reinstall existing tools across all your build and test servers for build agents to use. With a cloud native approach, teams can use this distribution mechanism to deliver and update tools across the organization. 
+
+An integrated CI/CD system also enables operations processes to be reliably automated. Operations activities like applying patches, security fixes, and maintenance can leverage the integrated CI system to provide updated containers for deployment.
+
+A sample Tekton pipeline: 
+
+image::/img/blog/Tekton-Intro.png[link="/img/blogs/Tekton-Intro.png" alt="Tekton Intro"]
+
+
+While Tekton is still new, the community of innovators that are contributing to and using Tekton is growing. Through layered steps and tasks, Tekton pipelines give flexibility to enterprises for their continuous integration and delivery needs. Tekton pipelines also allow enterprises to meet their customer demands by quickly delivering vital application updates that satisfy enterprise-specific quality and security requirements. 
+
+ 
+To see Tekton in action, go to the link:https://kabanero.io/[Kabanero website]. Kabanero brings together Knative, Istio, and Tekton, with Codewind and Appsody. Together, these open source projects provide an end-to-end solution for you to build, deploy, and manage the lifecycle of Kubernetes-based applications. 
```

Comment:
I feel like we need a better link. You can't really "see tekton in action" on kabanero.io.

On the other hand, maybe the text can just change to: "To get the latest news with Tekton in Kabanero, check out the..."

Reply:
also keep in mind the blog will be on kabanero.io, so maybe we can just delete that first sentence.